BUILD

```diff
@@ -34,11 +34,11 @@ load(
 )
 
 # This should be updated along with build.yaml
-g_stands_for = "gambit"
+g_stands_for = "generous"
 
 core_version = "4.0.0-dev"
```

Comment:
For consistency, could you update core to 5.0.0-dev please ?

Reply:
I didn't change it in either file, and yet they have different numbers. Are you sure `5.0.0-dev` is the correct value?